lazer/sdk/js/src/util/buffer-util.ts

```diff
@@ -0,0 +1,42 @@
+// the linting rules don't allow importing anything that might clash with
+// a global, top-level import. we disable this rule because we need this
+// imported from our installed dependency
+// eslint-disable-next-line unicorn/prefer-node-protocol
+import { Buffer as BrowserBuffer } from "buffer";
+
+import type { Data } from "isomorphic-ws";
+
+const { Buffer: PossibleBuiltInBuffer } = globalThis as Partial<{
+  Buffer: typeof Buffer;
+}>;
+
+const BufferClassToUse = PossibleBuiltInBuffer ?? BrowserBuffer;
+
+export class IsomorphicBuffer extends BufferClassToUse {
```

Comment:
I don't think there's any good reason to use an es6 class here -- there is no state being captured and only a static method here.   Conceptually you are only exporting functions that have no instances or state, but if this were to get extended, then it would degrade the quality of tree-shaking since many bundlers cannot tree shake parts of es6 classes at all, and even those that can have very major limitations in how they can do it.

I would just go back to what you had before and export `fromWebsocketData` as a naked function not encapsulated in a class.

Reply:
also done in 7aebe1f1b2c2709ca9de50a2e0b09efc8496d4b5

great suggestion, btw